.dotnet/src/Custom/Images/ImageGenerationOptions.cs

```diff
@@ -33,7 +33,7 @@ public partial class ImageGenerationOptions
     /// </item>
     /// </list>
     /// </summary>
-    public ImageResponseFormat? ResponseFormat { get; set; }
+    public ImageResponseFormat? ResponseFormat { get; init; }
```

Comment:
Should this also be "GeneratedImageResponseFormat" ? (alternatively, "GeneratedImageFormat")

Reply:
I like that -- I'll do `GeneratedImageFormat` since the "response" feels implied enough.